### DISCUSSION TITLE
Port to Synology NAS

### DISCUSSION THREAD

**Discussion by JohnnyDMonic** [hsoft/dupeguru#1032] [Ideas]
I just bought a synology NAS and their duplicate file detection sucks.

It would be awesome to have dupeguru as an installable package on the NAS itself, rather than have to run it on a PC connected to the NAS via SMB shares.

https://www.synology.com/en-us/support/developer

Just a thought.

**Comment by jeffkruczynski** [comment]
You can use https://github.com/jlesage/docker-dupeguru/releases. I have it installed on my DS916+ and it works great.
